stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -59,14 +59,17 @@ public type Client client object {
     #
     # + sqlQuery - SQL statement to execute
     # + parameters - The parameters to be passed to the update query
+    # + returnGeneratedKeys - Indicate that generated keys should be made available for retrieval. If the value is
+    #                         set to `true`, auto-generated keys are made available to retrieve when the statement
+    #                         is executed. If the underline JDBC driver does not support it, it will return an `Error`.
     # + return - `UpdateResult` with the updated row count and key column values,
     #             else `Error` will be returned if there is an error
-    public remote function update(@untainted string sqlQuery, Param... parameters)
-                                  returns UpdateResult|Error {
+    public remote function update(@untainted string sqlQuery, boolean returnGeneratedKeys, Param... parameters)
```

Comment:
Can't we make this a defaultable parameter with default value is `false`. Then we don't need to append `false` for simple update statements which doesn't have parameters